frontend/components/Header/Nav/MainMenu/Column.tsx

```diff
@@ -100,7 +100,7 @@ const ColumnLink: React.SFC<{
         role="none"
     >
         <a className={cx(columnLinkTitle)} href={link.url} role="menuitem">
-            {link.title}
+            {link.longTitle || link.title}
```

Comment:
Does this logic belong in `parse-capi`, I wonder? I'm thinking that in the presentation, the link should display whatever the link is told to display, according to the data model.

Reply:
True, keeping logic like this out of the rendering logic where possible should be a rule of thumb, even simple checks like this.